src/lib/common/limits.h

```diff
@@ -57,11 +56,23 @@
 #define SERVICE_PATH_MAX_COMPONENT_LEN    50
 #define SERVICE_PATH_MAX_TOTAL            (((SERVICE_PATH_MAX_COMPONENT_LEN + 1) * SERVICE_PATH_MAX_LEVELS) + 2) * SERVICE_PATH_MAX_COMPONENTS
 
-/* ****'************************************************************************
+
+
+/* ****************************************************************************
 *
 * Others -
 *
 */
-#define IP_LENGTH_MAX  15    // Based in xxx.xxx.xxx.xxx
+#define IP_LENGTH_MAX        15    // Based on xxx.xxx.xxx.xxx
+#define STRING_SIZE_FOR_INT  16    // Room enough for an integer
+
+
+
+/* ****************************************************************************
+*
+* Alarm Manager definitions - 
+*/
+#define ALARM_MGR_NOTIFICATION_ERROR_LOG_SAMPLING     50
```

Comment:
To have a behaviour similar to the existing one in 0.26.1, this (and the next) should be 1.